rustls/src/server/server_conn.rs

```diff
@@ -810,6 +810,9 @@ mod connection {
         }
 
         /// Send the alert to the client.
+        ///
+        /// To account for short-writes this function should be called repeatedly until it
```

Comment:
The use of a hyphen in "short-writes" seems surprising to me? I'd write it as two words (but you are the native speaker here, so happy to defer to your judgement).

Reply:
>  I'd write it as two words 

Yeah, agreed. Not sure why I had it with a hyphen :laughing: 

Fixed!